### ISSUE TITLE
Incompatible with Django 3

### ISSUE THREAD

**Issue by Tecktron** [jimfunk/django-postgresql-netfields#98]
`'ENGINE': 'django.db.backends.postgresql_psycopg2',` is deprecated and has been removed in Django 3 making this no longer compatible with Django 3.

There's an open PR open to fix the issue: 
https://github.com/jimfunk/django-postgresql-netfields/pull/96

**Comment by jimfunk** [comment]
This has just now been merged and released as 1.2.0. Sorry about the delay.

**Comment by jaychoo** [comment]
@jimfunk another Django 3 compatibility issue

```
from django.utils.six import with_metaclass, text_type
--
120 | ModuleNotFoundError: No module named 'django.utils.six'
```
